src/sentry/mail/activity/base.py

```diff
@@ -237,35 +280,27 @@ def send(self):
         email_type = self.get_email_type()
         headers = self.get_headers()
 
-        for user, reason in participants.items():
-            if group:
-                context.update(
-                    {
-                        "reason": GroupSubscriptionReason.descriptions.get(
-                            reason, "are subscribed to this issue"
-                        ),
-                        "unsubscribe_link": generate_signed_link(
-                            user.id,
-                            "sentry-account-email-unsubscribe-issue",
-                            kwargs={"issue_id": group.id},
-                        ),
-                    }
-                )
-            user_context = self.get_user_context(user)
-            if user_context:
-                user_context.update(context)
-            else:
-                user_context = context
-
-            msg = MessageBuilder(
-                subject=self.get_subject_with_prefix(),
-                template=template,
-                html_template=html_template,
-                headers=headers,
-                type=email_type,
-                context=user_context,
-                reference=activity,
-                reply_reference=group,
-            )
-            msg.add_users([user.id], project=project)
-            msg.send_async()
+        for provider, mapping in participants.items():
+            if provider == ExternalProviders.SLACK:
+                integrations = get_integrations(organization, ExternalProviders.SLACK)
+                for integration in integrations:
+                    for user, reason in mapping.items():
+                        send_slack_message_to_user(
```

Comment:
I'm not totally crazy about a module named `mail.activity` being the one conditionally triggering slack notifications.

But I understand it might be a lot of work to refactor the notification dispatch system